EIPS/eip-7612.md

```diff
@@ -45,43 +45,35 @@ def is_fork_block(block):
 # Write an account in the verkle tree
 def verkle_set_account(tree: VerkleTree, key: Bytes32, account: Optional[Account]):
     if account is not None:
-        versionkey = key
-        tree.set(versionkey, 0)
-        balancekey = key
-        balancekey[31] = BALANCE_LEAF_KEY
-        tree.set(balancekey, account.balance)
-        noncekey = key
-        noncekey[31] = NONCE_LEAF_KEY
-        tree.set(noncekey, account.nonce)
+        basicdata = bytes(0) # Version
+        basicdata += bytes(4) # Reserved
+        basicdata += len(account.code).to_bytes(3, 'big')
+        basicdata += account.nonce.to_bytes(8, 'big')
+        basicdata += account.balance.to_bytes(16, 'big')
```

Comment:
Update to do `BASIC_DATA` packing. This code assumes [this PR](https://github.com/ethereum/EIPs/pull/8753) will be accepted where we switch to big-endian encoding.